backend/routes/user_routes.py

```diff
@@ -0,0 +1,51 @@
+from os import environ
+from fastapi import APIRouter, HTTPException, Depends
+from sqlmodel import Session, select
+from uuid import uuid4
+from datetime import timedelta
+
+from models import User
+from core.database import get_session
+from utils.jwt_handler import create_jwt_token
+
+router = APIRouter()
+webdomain = environ.get("WEBDOMAIN")
+
+
+@router.post("/login")
+async def send_login_link(
+    phone_number: int, session: Session = Depends(get_session)
+):
+    user = session.exec(
+        select(User).where(User.phone_number == phone_number)
+    ).first()
+    if not user:
+        raise HTTPException(status_code=404, detail="User not found")
```

Comment:
As a first implementation, we can create the user here and send them a login link. That way we have a trivial onboarding. We will have to keep an eye on the registrations, to check that no-one will try anything funny, but we won't need an admin interface until much later into the project. Until then, PGAdmin will serve as our admin interface.

Reply:
solved by database seeder CLI